lib/tasks/data/truncate_data.rake

```diff
@@ -1,81 +1,44 @@
+# frozen_string_literal: true
+
+require 'highline'
+require 'tasks/data/truncate_data'
+
 # This task can be used to significantly reduce the size of a database
 #   This is used for example when loading live data into a staging server
 #   This way the staging server is not overloaded with too much data
+#
+# This is also aimed at implementing data archiving. We assume data older than
+# 2 years can be safely removed and restored from a backup. This gives room for
+# hubs to do their tax declaration.
+#
+# It's a must to perform a backup right before executing this. Then, to allow
+# for a later data recovery we need to keep track of the exact moment this rake
+# task was executed.
+#
+# Execute this in production only when the instance users are sleeping to avoid any trouble.
+#
+# Example:
+#
+# $ bundle exec rake "ofn:data:truncate[12]"
```

Comment:
I'm thinking it's safer to make the 2-years window the default instead of the current 3 months. Thoughts?

Reply:
:+1:

Reply:
2 years would be 24 here, right?

Reply:
Oh shit. I need to rest, yes :joy: